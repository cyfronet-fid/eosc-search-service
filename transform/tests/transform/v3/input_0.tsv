50|tubitakulakb::5320f0701ac3937aef1a5c6f1dce7c0bOkul Yöneticilerinin Atanmasında Kaotik Dönem ve Etkileri. Bu çalışmada 2014 yılında okul müdürlerinin, müdürlük görevinden alınmaları ve yeniden müdür olarak görevlendirilmemelerine ilişkin süreçte yaşananlar ve bu durumla başlayan kaotik dönem tekrar görevlendirilmeyen müdürlerle yapılan görüşmelerden elde edilen bulgulara ve ulaşılan sonuçlara dayalı olarak incelenmektedir. Veriler nitel yollarla elde edilmiştir. Yarı yapılandırılmış görüşme formu kullanılarak 14 kişi ile yüz yüze görüşülmüş, veri kaybı olmaması için görüşmeler kayıt altına alınmıştır. Veri çözümleme Nvivo 10 yazılım programı kullanılarak gerçekleştirilmiştir. Araştırmanın katılımcıları; müdürlerin görevden alınma sürecini tamamen “öznel” bulduklarını, görevden alınmaların nesnel herhangi bir nedene dayanmadığını, aksine “politik” görüşleri nedeniyle “keyfiyet” ve “adam kayırmacılığa” dayalı olarak gerçekleştiğini belirtmiştir. Katılımcılar, bu sürecin genelde eğitim sistemi özelde de okullar, okul paydaşları, yeni görev verilen yöneticiler ve kendileri açısından oldukça olumsuz sonuçlar doğurduğunu ileri sürmüşlerdir. Araştırma konusu olan sürece ilişkin bulgular ve ulaşılan sonuçlar üzerinde yapılan yorum ve önerilerin, hem gelecekte yapılacak müdür atamalarına hem de bu alanda yapılacak tartışmalara katkıda bulunması umulmaktadır.Okul müdürü,Kaotik dönemSinan OLKUNOpen AccessUndetermined
                                
                                                                    Cilt: 16 Sayı: 32017-06-01Şahin, İdrisKesik, FATMABeycioğlu, Kadir\N\N
50|sarcservicod::d5a5a3d06964f39b26ab924279f57692Eventos: Taking Nature Seriously: Citizens, Science and EnvironmentEventos: Taking Nature Seriously: Citizens, Science and EnvironmentEventos: Taking Nature Seriously: Citizens, Science and EnvironmentEventos: Taking Nature Seriously: Citizens, Science and EnvironmentFundação Fernando Pessoa/Edições Universidade Fernando PessoaOpen AccessPortugueseAntropológicas2012-07-23Cabral, AlcindaMaia, Rui Leandro\N\N
50|orcid_______::49f97c9d10c29e53c581da6d01647dca\NLinear and non-linear features for intrapartum cardiotocography evaluation\N\N\Nnot availableUndetermined\N\N\NChudáček, V.Spilka, J.Huptych, M.Georgoulas, G.Lhotská, L.Stylios, C.Michal Koucký0000-0003-2807-7635Janků, P.\N\N
50|orcid_______::49f608129559ce580ff50bfa8a1ea847\NParasitismo en campo de Coccinella septempunctata (Coleoptera: Coccinellidae) por su enemigo natural Dinocampus sp Foerster, 1862 (Hymenoptera: Braconidae).\N\N\Nnot availableUndetermined\N\N\NAgustín Hernández Juárez0000-0001-7059-4471\N\N
50|arXiv_______::046fe50541eb606183642eea0bad910e1707.06207The cohomology of rank two stable bundle moduli: mod two nilpotency & skew Schur polynomialsWe compute cup product pairings in the integral cohomology ring of the moduli space of rank two stable bundles with odd determinant over a Riemann surface using methods of Zagier. The resulting formula is related to a generating function for certain skew Schur polynomials. As an application, we compute the nilpotency degree of a distinguished degree two generator in the mod two cohomology ring. We then give descriptions of the mod two cohomology rings in low genus, and describe the subrings invariant under the mapping class group action.Comment: 34 pages, 2 figuresMathematics - Geometric Topology14D20, 57R58\NOpen AccessEnglish\N2017-07-19Scaduto, ChristopherStoffregen, MatthewScaduto Christopher WScaduto Christopher W20|pending_org_::a1edd04703a88b3e487b1366875cb197PostDoctoral Research Fellowship40|nsf_________::b312745f8d888b21819df6a1f4d8cc281503100
50|od______1033::1137b783bd0d3ff5212c8860d83614edAnálisis del estándar Passivhaus en la climatología española"A partir del 1 de enero de 2018 en toda Europa todos los edificios de carácter público que sean de nueva construcción deberán cumplir la normativa europea nZEB (nearly Zero Energy Buildings, en español EECN, Edificios de Energía Casi Nula). Asimismo, a partir del 1 de enero de 2020 cualquier tipo de nueva construcción tendrá que adaptarse también” ¿Cómo llegar a cumplir con estas directrices? Parece un poco complicado teniendo en cuenta el tiempo que queda para alcanzar estas fechas, pero es posible. Según la PEP, Plataforma de Edificación Passivhaus en España, hay unos principios básicos sobre los que actuar para cumplir con esta definición. En este trabajo, el objetivo será estudiar y profundizar en dos de ellos: aquellos que permitan una mayor experimentación y variación en relación con la composición de los elementos constructivos del edificio. Profundizando en estos principios seleccionados se pretende detectar medidas y patrones para lograr la reducción de la demanda energética en edificaciones situadas en diversos climas dentro del territorio español, poniendo el foco en elementos de la envolvente del edificio. From January 1, 2018 in the whole Europe all the buildings of public character that are of new construction will have to fulfill with the European regulation building nZEB of almost void energy. Likewise, from January 1, 2020 any type of building, already be of new construction or of rehabilitation, it will have to adapt to this definition. How to manage to expire with these directives? It seems to be a bit complicated bearing in mind the time that stays, but it is possible. According to the PEP, Platform of Building Passivhaus in Spain, there are certain basic principles on which to affect to reach this definition. In this work, the goal is to deepen and study two of them: those that allow a greater experimentation and variation in relation to the composition of buildings constructive elements. By deepening on these selected principles it is tried to detect measures and patterns to achieve the reduction of the energetic demand in buildings located in diverse climates within the Spanish territory, focusing on building envelope elements.ArquitecturaDerechoE.T.S. Arquitectura (UPM)Open AccessSpanish; Castilian\N2018-06-01Burón Gutiérrez, PabloTechnical University of MadridUPM20|openorgs____::24c22d6273aeb0a220d328d8303e2772\N
