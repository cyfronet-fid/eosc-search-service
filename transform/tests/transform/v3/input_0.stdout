{"id": "50|sarcservicod::d5a5a3d06964f39b26ab924279f57692", "pid": [], "title": ["Eventos: Taking Nature Seriously: Citizens, Science and Environment", "Eventos: Taking Nature Seriously: Citizens, Science and Environment", "Eventos: Taking Nature Seriously: Citizens, Science and Environment", "Eventos: Taking Nature Seriously: Citizens, Science and Environment"], "description": [], "subject": [], "publisher": "Funda\u00e7\u00e3o Fernando Pessoa/Edi\u00e7\u00f5es Universidade Fernando Pessoa", "bestaccessright": ["Open Access"], "language": ["Portuguese"], "journal": "Antropol\u00f3gicas", "fulltext": null, "published": ["2012-07-23"], "resourceType": "conference poster", "subjects": ["keyword:reproductive health, students, contraception", "mesh:education"], "author_names": ["Cabral, Alcinda", "Maia, Rui Leandro"], "author_pids": ["", ""], "organization_names": [], "organization_shorts": [], "organization_ids": [], "project_titles": [], "project_ids": [], "project_codes": []}
{"id": "50|orcid_______::49f97c9d10c29e53c581da6d01647dca", "pid": [], "title": ["Linear and non-linear features for intrapartum cardiotocography evaluation"], "description": [], "subject": [], "publisher": null, "bestaccessright": ["not available"], "language": ["Undetermined"], "journal": null, "fulltext": null, "published": [], "resourceType": "conference poster", "subjects": ["keyword:reproductive health, students, contraception", "mesh:education"], "author_names": ["Chuda\u0301c\u030cek, V.", "Spilka, J.", "Huptych, M.", "Georgoulas, G.", "Lhotska\u0301, L.", "Stylios, C.", "Michal Kouck\u00fd", "Janku\u030a, P."], "author_pids": ["", "", "", "", "", "", "0000-0003-2807-7635", ""], "organization_names": [], "organization_shorts": [], "organization_ids": [], "project_titles": [], "project_ids": [], "project_codes": []}
{"id": "50|orcid_______::49f608129559ce580ff50bfa8a1ea847", "pid": [], "title": ["Parasitismo en campo de Coccinella septempunctata (Coleoptera: Coccinellidae) por su enemigo natural Dinocampus sp Foerster, 1862 (Hymenoptera: Braconidae)."], "description": [], "subject": [], "publisher": null, "bestaccessright": ["not available"], "language": ["Undetermined"], "journal": null, "fulltext": null, "published": [], "resourceType": "conference poster", "subjects": ["keyword:reproductive health, students, contraception", "mesh:education"], "author_names": ["Agust\u00edn Hern\u00e1ndez Ju\u00e1rez"], "author_pids": ["0000-0001-7059-4471"], "organization_names": [], "organization_shorts": [], "organization_ids": [], "project_titles": [], "project_ids": [], "project_codes": []}
{"id": "50|arXiv_______::046fe50541eb606183642eea0bad910e", "pid": ["1707.06207"], "title": ["The cohomology of rank two stable bundle moduli: mod two nilpotency & skew Schur polynomials"], "description": ["We compute cup product pairings in the integral cohomology ring of the moduli space of rank two stable bundles with odd determinant over a Riemann surface using methods of Zagier. The resulting formula is related to a generating function for certain skew Schur polynomials. As an application, we compute the nilpotency degree of a distinguished degree two generator in the mod two cohomology ring. We then give descriptions of the mod two cohomology rings in low genus, and describe the subrings invariant under the mapping class group action.", "Comment: 34 pages, 2 figures"], "subject": ["Mathematics - Geometric Topology", "14D20, 57R58"], "publisher": null, "bestaccessright": ["Open Access"], "language": ["English"], "journal": null, "fulltext": null, "published": ["2017-07-19"], "resourceType": "conference poster", "subjects": ["keyword:reproductive health, students, contraception", "mesh:education"], "author_names": ["Scaduto, Christopher", "Stoffregen, Matthew"], "author_pids": ["", ""], "organization_names": ["Scaduto Christopher W"], "organization_shorts": ["Scaduto Christopher W"], "organization_ids": ["20|pending_org_::a1edd04703a88b3e487b1366875cb197"], "project_titles": ["PostDoctoral Research Fellowship"], "project_ids": ["40|nsf_________::b312745f8d888b21819df6a1f4d8cc28"], "project_codes": ["1503100"]}
